### ISSUE TITLE
Expandable buttons should have aria-expanded attribute

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/nextcloud-vue#2755]
Accessibility issue.

All expandable buttons (mostly in a sidebar) should have `aria-expanded` attribute.

Screen reader users should be told of this function if the area is open or not.

![image](https://user-images.githubusercontent.com/6078378/173071258-f31e01c3-56e1-42b5-a759-3026aff59c6b.png)

![image](https://user-images.githubusercontent.com/6078378/173071307-d6c1d2ad-e8ac-4a14-ad0c-b6da4974dbfa.png)
